registry/tracecat_registry/base/core/http.py

```diff
@@ -72,7 +97,33 @@ async def http_request(
     """Perform a HTTP request to a given URL."""
 
     try:
+        cert = None
+        if secrets.get("SSL_CLIENT_CERT"):
+            # Create a temp file for the certificate
+            cert_file_path = None
+            with tempfile.NamedTemporaryFile(delete=False) as cert_file:
+                cert_file.write(secrets.get("SSL_CLIENT_CERT").encode())
+                cert_file.flush()
+                cert_file_path = cert_file.name
+
+            # Create a temp file for the key (if exists)
+            key_file_path = None
+            if secrets.get("SSL_CLIENT_KEY"):
+                with tempfile.NamedTemporaryFile(delete=False) as key_file:
+                    key_file.write(secrets.get("SSL_CLIENT_KEY").encode())
+                    key_file.flush()
+                    key_file_path = key_file.name
+
+            cert = [
+                cert_file_path,
+                key_file_path,
+                secrets.get("SSL_CLIENT_PASSWORD"),
+            ]
+            # Drop None values
+            cert = tuple([c for c in cert if c is not None])
```

Comment:
unnecessary listcomp
```suggestion
            cert = tuple(c for c in cert if c is not None)
```